### PR TITLE
✨ [tuning] Added "Clear Queue" button and a new random footer

### PR DESCRIPTION
**Pull request by furtidev**
### Things changed:

- [x] Added a clear queue button to the `QueueCommandView`
- [x] Added a new random footer.

### CODE REVIEW

File: cogs/music.py [hunk 1]
```diff
@@ -446,6 +446,16 @@ async def previous(self, _: disnake.ui.Button, inter: disnake.MessageInteraction
             view=self,
         )
 
+    @disnake.ui.button(label='Clear Queue!', style=disnake.ButtonStyle.danger)
+    async def clear_queue(self, _: disnake.ui.Button, inter: disnake.MessageInteraction) -> None:
+        self.inter.voice_state.songs.clear()
+
+        await inter.response.edit_message(
+            "Your queue has been cleared!",
```
Comment (hitblast) [inline cogs/music.py:454]: Could you remove the string normalization here? Like just use `'` instead of `"`? It's not a biggie, though.

**Review by hitblast — APPROVED** [review]
LGTM!

File: cogs/music.py (hunk 1 above)
Comment (furtidev) [reply cogs/music.py:454]: Oh, sorry about that. Fixing.

Comment (hitblast) [reply cogs/music.py:454]: @furtidev As I said, not a biggie! :)

**Review by hitblast — COMMENTED** [review]
Approved!

**Pull request merged**